pkg/web_css/lib/src/_list.scss

```diff
@@ -241,6 +241,10 @@
     }
   }
 
+  .topics-tag {
+  padding: 4px 4px;
```

Comment:
Let's also add `display: inline-block`, otherwise I'm not sure if the padding will get applied (or maybe it will, I'm just not sure).